node/src/components/fetcher.rs

```diff
@@ -351,7 +351,7 @@ impl ItemFetcher<BlockWithMetadata> for Fetcher<BlockWithMetadata> {
         peer: NodeId,
     ) -> Effects<Event<BlockWithMetadata>> {
         effect_builder
-            .get_block_at_height_with_metadata_from_storage(id)
+            .get_block_and_sufficient_finality_signatures_by_height_from_storage(id)
```

Comment:
Maybe we could introduce the term "fully signed block" or even "signed block" for "block with sufficient finality signatures"
→ `get_signed_block_by_height_from_storage`?

Reply:
Sure!  But I think we should fix this in another PR...